docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/CubeDockerConfiguration.java

```diff
@@ -307,4 +315,24 @@ private static String resolveFilename(URI uri) {
 
         return content.toString();
     }
+
+    public static String resolveUrl(String machineVersion) {
+        validation(machineVersion, DOCKER_MACHINE_VERSION);
```

Comment:
This force the user to configure a docker-machine version? Any URL that could support 'latest'?